cmd/importcorpora/main.go

```diff
@@ -0,0 +1,203 @@
+// Usage: go run cmd/importcorpora/main.go
+//
+// Updates the at the bottom of this file specified data files with content from dariusk/corpora.
+package main
```

Comment:
I noted down the makefile comment somewhere and then forgot to add. Completely agree! 👍